sdk/core/core-client/CHANGELOG.md

```diff
@@ -1,7 +1,12 @@
 # Release History
 
-## 1.0.0 (Unreleased)
+## 1.0.0 (2021-03-15)
 
+GA release of this package.
+
+- Fix serialization of body of false ([#14218](https://github.com/Azure/azure-sdk-for-js/pull/14218))
+- Set custom headers after serializing headers ([#14268](https://github.com/Azure/azure-sdk-for-js/pull/14268))
+- Allow setting empty query parameter values ([#14260](https://github.com/Azure/azure-sdk-for-js/pull/14260))
```

Comment:
these are all good fixes, but since no public interface changed, do we want to document them here?

Reply:
We document bug fixes in changelog, so am good with this. Helps one to understand what they stand to gain going from one version to another

Reply:
oops I just removed them.  I think they are mostly for us than end users.